index.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
 
   env: {
     es6: true,
+    jest: true,
```

Comment:
Assume you are using this for a new project?
No way to specify this in the project itself?